windows/RNGetRandomValues61/RNGetRandomValues61.vcxproj

```diff
@@ -0,0 +1,166 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can you elaborate on the windows/RNGetRandomValues vs windows/RNGetRandomValues61 ?

Reply:
Unfortunately, 61-62 involved several tooling changes, and it would be a little complicated to have one project file that supported both. So even though the source files can be shared, having two projects, one for 61, and one for 62+ is a workaround pattern.

Going forward, especially once @jfkm69 switches to consuming the CppLib props/targets, will make the regular RNGetRandomValues.vcxproj better insulated against future changes to RNW.